jobs/utilityapi_software_engineer.html

```diff
@@ -17,33 +17,40 @@
   - united-states
```

Comment:
United Staes shouldn't be a tag. This field is reserved for technical skills required for the job.

Reply:
Fixed!